src/test/java/io/confluent/connect/elasticsearch/DataConverterTest.java

```diff
@@ -239,4 +245,63 @@ public void struct() {
     );
   }
 
+  @Test
+  public void ignoreNull() {
+    final String key = "key";
+    final String topic = "topic";
+    final int partition = 12;
+    final long offset = 26;
+    final String index = "index";
+    final String type = "type";
+
+    converter = new DataConverter(true, BehaviorOnNullValues.IGNORE);
+
+    Schema schema = SchemaBuilder.struct().name("struct").field("string", Schema.STRING_SCHEMA).build();
+    SinkRecord sinkRecord = new SinkRecord(topic, partition, Schema.STRING_SCHEMA, key, schema, null, offset);
```

Comment:
Probably could make the `schema` a static field, and could even add a method that created the `SinkRecord` instance that took just the value parameter. The test then becomes something like:

```
@Test
public void ignoreNull() {
  converter = new DataConverter(true, BehaviorOnNullValues.IGNORE);
  record = createRecordWithValue(null);
  assertNull(converter.convertRecord(record, index, type, false, false));
}
```

IMO, it's often much better to have a simple test method that is very easy to see what it is doing, especially when there are other tests doing similar things.

Reply:
Totally up for that ^^

Reply:
I like that approach, will try to adopt it appropriately in the next commit.